tests/elm.test.zsh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env zsh +x
```

Comment:
shebang treats arguments differently on various operating systems. [Linux kernel treats everything following the first “word” in the shebang line as a single argument.](https://lwn.net/Articles/630727/). More information on this [SO answer](https://stackoverflow.com/a/4304187)

**More**:
- [Details about the shebang/hash-bang mechanism on various Unix flavors](https://www.in-ulm.de/~mascheck/various/shebang/)

Reply:
Good to know. I copied from `tests/char.test.zsh`, so it might need to be changed there too.